test/system/markdown_test.rb

```diff
@@ -4,6 +4,8 @@
 
 class MarkdownTest < ApplicationSystemTestCase
   test 'speak block test' do
+    user = users(:mentormentaro)
+    reset_avatar(user)
```

Comment:
```suggestion
    reset_avatar(users(:mentormentaro))
```

1箇所でしか出てこないのであればこれでいいかもしれません。また、複数のテストケースで出てくるのであれば`setup`ブロックで共有のユーザーとしてインスタンス変数に入れるのも良いかもしれません。